app/views/devise/registrations/edit.html.haml

```diff
@@ -2,42 +2,72 @@
   %h1.entry-title= t('.title')
 .post-title-divider
   %span
-.entry-content.wpcf7
+.entry-content
   = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
 
     != model_errors_helper(resource)
 
-    .field
-      = f.label :first_name
-      = f.text_field :first_name, autofocus: true, class: 'wpcf7-form-control'
-    .field
-      = f.label :last_name
-      = f.text_field :last_name, class: 'wpcf7-form-control'
-    .field
-      = f.label :email
-      = f.email_field :email, class: 'wpcf7-form-control'
-    - if devise_mapping.confirmable? && resource.pending_reconfirmation?
-      %div
-        "#{t('.unconfirmed_email_for', unconfirmed_email: resource.unconfirmed_email )}"
-    .field
-      = f.label :password
-      %i= t('.leave_blank_if_no_change')
-      = f.password_field :password, autocomplete: 'off', class: 'wpcf7-form-control'
-      = render 'devise/shared/min_password_length_info'
-    .field
-      = f.label :password_confirmation, "#{t('.confirm_password')}"
-      = f.password_field :password_confirmation, autocomplete: 'off', class: 'wpcf7-form-control'
-    .field
-      = f.label :current_password, "#{t('.current_password')}", class: 'required'
-      %i= t('.required_to_save_changes')
-      = f.password_field :current_password, autocomplete: 'off', class: 'wpcf7-form-control'
-    .actions
-      = f.submit "#{t('.submit_button_label')}"
-  = render 'application/required_fields'
-
-  %h2.entry-title= t('.unregister')
-  .post-title-divider
-    %span
-  %p
-    #{button_to t('.delete_my_account'), registration_path(resource_name), class:'btn btn-danger',  data: { confirm: t('.confirm_are_you_sure') }, method: :delete}
-  = link_to "#{t('.back')}", :back
+    - if resource.member?
+
+      .col-sm-3
```

Comment:
pull out the photo-related code into a partial.  

(This file is already too long as we have it.)

Reply:
Done.